### PR TITLE
fixed look and feels of genome widget a bit

### PR DESCRIPTION
**Pull request by nathandunn**
Fixes AGR-310.

### CODE REVIEW

**Comment by nathandunn** [comment]
Let me know if you have any UI recommendations for those buttons.

screen shot 2017-06-05 at 10 35 23 am

**Comment by paaatrick** [comment]
I was going to ask what the purpose of those links/buttons is in the first place? The first one takes you to the same place as just clicking the image. And the second one just gives you the same (maybe slightly wider) image again? I guess my point is the first one seems redundant and I'm not sure how valuable the second one is. 

But if you think they're important to keep, yeah, I would tone it down a bit. I think `btn-secondary` would be more appropriate than `btn-warning`. I'd take off the icons, too. Maybe even move them below the image, since they seem of secondary importance to the image itself?

Regarding the section name change, someone else requested it be called "Gene Model" (https://agr-jira.atlassian.net/browse/AGR-436)

File: webapp/src/containers/genePage/transcriptInlineViewer.js [hunk 1]
```diff
@@ -46,8 +46,10 @@ class TranscriptViewer extends Component {
     return (
       <div id="genomeViewer">
         {/*<iframe id="genomeFrame" className={style.jbrowse} src={externalJbrowseUrl}/>*/}
-        <a href={externalJbrowseUrl.replace('overview.html','index.html')}>Genome Viewer<i className="fa fa-link" /> </a>
-        <a href={externalJbrowseUrl}>Overview<i className="fa fa-link" /> </a>
+        <a href={externalJbrowseUrl.replace('overview.html','index.html')} className="btn btn-warning btn-sm">Browse <i className="fa fa-search" /></a>
+        &nbsp;
+        &nbsp;
```
Comment (paaatrick) [inline webapp/src/containers/genePage/transcriptInlineViewer.js]: Are these just to add space between the buttons? Can that be done with CSS instead if needed?

**Comment by nathandunn** [comment]
Fixed:
- changed it to "Gene Mode" for https://agr-jira.atlassian.net/browse/AGR-436
- made secondary buttons (for some reason btn-default wasn't working), got rid of ```&nbsp;```

I left the buttons on there even though the functionality is duplicate.  Mostly because there really aren't any directions associated with it and the image itself doesn't imply a link for the user.   I tried a blue background for the image, but of course that looks horrible as well.   When we move to divs, I think that will help.

**Comment by nathandunn** [comment]
Oh, I removed the snapshot button.   It was mostly there for dev purposes.

**Pull request merged**